nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/java/org/apache/nifi/atlas/processors/EgressProcessors.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.atlas.processors;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EgressProcessors {
+
+    private static final Map<String, Egress> processors = new HashMap<>();
+
+    static {
+        processors.put("org.apache.nifi.processors.hive.PutHiveStreaming", new PutHiveStreaming());
```

Comment:
So these Egress/Ingress processors are all that would be supported initially I take it?  It will be interesting to see how this works out and then perhaps we can have this done via an Annotation on processors that could be supported for egress/ingress.